### PR TITLE
[fix bug 1132289] Plugin check minify JS error

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@pmclanahan @schalkneethling r?

**Comment by schalkneethling** [comment]
lgtm r+ :beer:

File: bedrock/mozorg/templates/mozorg/plugincheck.html [hunk 1]
```diff
@@ -215,5 +215,6 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
 {% endblock %}
 
 {% block js %}
```
Comment (pmclanahan) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: Please add a brief comment here about why this was required, and note that we only need it until we get the unminified source.

**Comment by pmclanahan** [comment]
r+wc :cake:

**Pull request merged**